static/app/views/discover/results.spec.tsx

```diff
@@ -1081,11 +1081,7 @@ describe('Results', function () {
         {context: initialData.routerContext, organization}
       );
 
-      await waitFor(() => {
```

Comment:
No need to await as the test will fail anyways as the tips are part of the response. We can instead just await tip to be in the document